Engine/EditorUI/Panel/PanelScene.cpp

```diff
@@ -135,6 +136,10 @@ void PanelScene::RenderSceneBar()
 			App->debug->show_debug_metrics = !App->debug->show_debug_metrics;
 		}
 
+		if (ImGui::Selectable("Reload shaders", false, ImGuiSelectableFlags_None, ImVec2(100, 0)))
+		{
+			App->program->LoadPrograms(SHADERS_PATH);
+		}
```

Comment:
Doesn't work :/

Reply:
how did you try it?

Reply:
i changed some code on the FS and then pressed the button